src/publishing/s3publisher.py

```diff
@@ -140,11 +141,14 @@ def publish_to_s3(directory, base_url, site_prefix, bucket, federalist_config,
     # Create lists of all the new and modified objects
     new_objects = []
     replacement_objects = []
+    # track whether we can do diffing because of cache control
+    default_cache_control = getenv('CACHE_CONTROL', 'max-age=60')
     for local_filename, local_obj in local_objects_by_filename.items():
         matching_remote_obj = remote_objects_by_filename.get(local_filename)
         if not matching_remote_obj:
             new_objects.append(local_obj)
-        else:
+        elif (matching_remote_obj.md5 != local_obj.md5 or
+              local_obj.cache_control != default_cache_control):
             replacement_objects.append(local_obj)
 
```

Comment:
Should we add an explicit `continue` statement to move onto the next iteration if either of the if conditions are not met?

Reply:
I'm not sure I understand this one. After the `if` statements, we're at the end of the loop anyway and will continue by default

Reply:
Sounds good to me.